### PR TITLE
Allocation2

### LINKED ISSUE #40: Add 2nd Allocation
Here is the response from allocations request (listing a new vesting contract)

https://safe-claiming-app-data.safe.global/allocations/1/<SAFEADDRESS>.json

(It now contains allocation for user_v2).

Related to this SEP: https://snapshot.org/#/safe.eth/proposal/0xb4765551b4814b592d02ce67de05527ac1d2b88a8c814c4346ecc0c947c9b941

### PR DESCRIPTION
**Pull request by bh2smith**
Closes #40

### CODE REVIEW

**Comment by josojo** [comment]
Hey @bh2smith. Thanks for providing this piece of code!

But unfortunately, it does not work out for me:
IF I am running the command FullClaim, then the redeem in the old contract fails -> Since redeeming was only possible, until time x.
IF I am running the command CLAIM, then the claiming works well, but on the new contract: 0xc0fde70a65c7569fe919be57492228dee8cdb585, 

Given that it seems we need to a mix from claiming and redeeming:
image
I think one needs to modificate the code even further. Or did this work for you?

Should I help or will you tackle it?

**Comment by bh2smith** [comment]
I believe I manually modified it to exclude some parts that didn't work. At the time I was only concerned with one time use. Maybe since the V1 claim period is over we can simply remove it permanently.

File: src/airdrop/tx.py [hunk 1]
```diff
@@ -19,32 +19,41 @@ def transactions_for(
...
+        for child, allocation_list in allocations.items():
+            transactions += [
+                build_and_sign_redeem(
+                    safe=parent,
+                    sub_safe=child,
+                    allocation=allocation,
+                )
+                for allocation in allocation_list
+                # all other tags are beyond the claim period
+                if allocation.tag == "user_v2"
```
Comment (bh2smith) [inline src/airdrop/tx.py:40]: This is likely where you were experiencing issues @josojo. try it now.

**Comment by josojo** [comment]
Nice, it worked Ben. Thanks

**Pull request merged**